File: app/src/main/java/com/liweiyap/narradir/NarradirApplication.java
package com.liweiyap.narradir;

import android.app.Application;

import com.liweiyap.narradir.util.SamsungSemEmergencyManagerLeakFixActivity;

public class NarradirApplication extends Application
{
    @Override
    public void onCreate()
    {
        super.onCreate();

        SamsungSemEmergencyManagerLeakFixActivity.applyFix(this);
    }
}

File: app/src/main/java/com/liweiyap/narradir/NarradirApplication.kt
package com.liweiyap.narradir

import android.app.Application

import com.liweiyap.narradir.util.SamsungSemEmergencyManagerLeakFixActivity

class NarradirApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        SamsungSemEmergencyManagerLeakFixActivity.applyFix(this)
    }
}